build/vagrant/build-spel-vagrant.pkr.hcl

```diff
@@ -0,0 +1,110 @@
+variable "aws_instance_type" {
+  type    = string
+  default = "c5n.metal"
+}
+
+variable "aws_temporary_security_group_source_cidrs" {
+  type    = list(string)
+  default = ["0.0.0.0/0"]
+}
+
+variable "spel_ci" {
+  type    = bool
+  default = false
+}
+
+variable "spel_identifier" {
+  type = string
+}
+
+variable "spel_packer_url" {
+  type = string
+}
+
+variable "spel_repo_commit" {
+  type    = string
+  default = "master"
+}
+
+variable "spel_repo_url" {
+  type    = string
+  default = "https://github.com/plus3it/spel.git"
+}
+
+variable "spel_version" {
+  type = string
+}
+
+variable "vagrant_cloud_token" {
+  type    = string
+  default = env("VAGRANT_CLOUD_TOKEN")
+}
+
+variable "vagrant_cloud_user" {
+  type    = string
+  default = "plus3it"
+}
+
+variable "virtualbox_iso_url_centos7" {
+  type = string
+}
+
+source "amazon-ebs" "ubuntu" {
+  ami_name                    = "builder-${var.spel_identifier}-vagrant-${var.spel_version}.x86_64-gp2"
+  associate_public_ip_address = true
+  communicator                = "ssh"
+  force_deregister            = true
+  instance_type               = var.aws_instance_type
+  launch_block_device_mappings {
+    delete_on_termination = true
+    device_name           = "/dev/sda1"
+    volume_size           = 8
+    volume_type           = "gp2"
+  }
+  max_retries            = 20
+  skip_create_ami        = true
+  skip_save_build_region = true
+  source_ami_filter {
+    filters = {
+      architecture        = "x86_64"
+      name                = "ubuntu/images/hvm-ssd/ubuntu-*-20.04-amd64-server-*"
```

Comment:
As written, no, this will not support building vagrant outside of the commercial partition. We could expose these values as variables, if we need to. But we haven't been building vagrant boxes anywhere else, as there is no vagrant cloud to push them to anywhere else. And no metal instances anywhere else, which are needed to run virtualbox in ec2. So using the template elsewhere is not yet feasible anyway.